deploy/parameter/helm-values.yaml

```diff
@@ -41,18 +41,31 @@ securityContext:
 
 resources: 
   limits:
-    cpu: 500m
+    cpu: 100m
     memory: 128Mi
```

Comment:
Seems double the request memory is too conservative. How about using 256Mi as a limit?

Reply:
Updated